ios/KaMPStarteriOS/BreedCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  BreedCell.swift
+//  KaMPStarteriOS
+//
+//  Created by Ben Whitley on 6/17/20.
+//  Copyright © 2020 Touchlab. All rights reserved.
+//
+
+import UIKit
+import shared
+
+protocol BreedCellDelegate: class {
+    func toggleFavorite(_ breed: Breed)
+}
+
+class BreedCell: UITableViewCell {
+    
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var favoriteButton: UIButton!
+    
+    var breed: Breed?
+    var delegate: BreedCellDelegate?
+    
+    func bind(_ breed: Breed) {
```

Comment:
This is mostly my Android thinking, and I know it was already like this, but is there any leak risk to not unbinding ever?

Reply:
It's good form to make the delegate a `weak var`, so thanks for pointing that out, but other than that, no. Each cell being initialized with `dequeueReusableCell(withIdentifier: , for: )` means that iOS will basically handle init/deinit of cells, as well as reusing cells if there are enough items to necessitate that. I'm sure there's times that are exceptions and it's possible to get really weird with it, but 99% of the time it'll be handled for you.